components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -16111,6 +16091,9 @@ public void addAPIRevision(APIRevision apiRevision) throws APIManagementExceptio
                             // Adding product id to uri template id just to store value
                             uriTemplate.setId(rs.getInt(7));
                         }
+                        //Add Operation Endpoint Mapping to URI TEMPLATE
+                        setOperationEndpointMapping(uriTemplate, rs.getInt("URL_MAPPING_ID"));
```

Comment:
Can we make this a constant? "URL_MAPPING_ID"

Reply:
This is the column name of the URL_MAPPING table in DB right?, So, I think no need to make this a constant.